static/app.scss

```diff
@@ -1,14 +1,115 @@
-@import "ams";
-@import "ams-bootstrap";
-@import "ams-ois";
+//
+//  Application specific styling
+//  specific
+//
 
-.borderless tbody td, .borderless thead th {
-  border: 0;
+@import "~stijl/src/styles/ams-stijl";
+
+
+.ams-header {
+  margin-top: 1rem;
+  margin-bottom: 1rem;
+  margin-left: -2rem;
+
+  &__logo {
+    &--short {
+      width: 150px;
+      height: 100px;
+    }
+  }
 }
 
-.chart-container {
-  height: 100%;
-  display: flex;
-  align-items: center;
-  justify-content: center;
+.sticky-top {
+  position: sticky;
+  top: 0;
+  background-color: $ams-wit;
+  box-shadow: 0 1px 1px 0 $ams-lichtgrijs;
+  z-index: 9999;
+}
+
+.page-overlay {
+  position: fixed;
+  top: 0;
+  left: 0;
+  width: 100vw;
+  height: 100vh;
+  background-color: rgba(0,0,0,.4);
```

Comment:
Does any ams color fit?

Reply:
no, as they are all hex color (which don't support transparancy, will add it to the stijl repo though)